octoprint_mrbeam/analytics/uploader.py

```diff
@@ -223,7 +223,7 @@ def upload_file(self, my_file, token_data):
 				err = "upload_file failed: {}".format(r.status_code)
 				self.set_status(my_file, succ=False, err=err)
 		except Exception as e:
-			self._logger.exception("Exception while upload_file %s:", my_file)
+			self._logger.exception("Exception while upload_file {file}: {err}", file=my_file, err=e)
```

Comment:
does this worK without the command format() ?

Reply:
Oops. Fixed!